service/src/main/java/dk/dbc/search/solrdocstore/ResourceBean.java

```diff
@@ -54,6 +59,15 @@ public Response addResource(String jsonContent) throws JSONBException {
         // Add resource
         BibliographicResourceEntity resource = request.asBibliographicResource();
         entityManager.merge(resource);
+        // Enqueue all related bib items
+        TypedQuery<BibliographicEntity> query = entityManager.createQuery(
+                "SELECT b FROM BibliographicEntity  b WHERE b.agencyId=:agencyId " +
+                        "AND b.bibliographicRecordId=:recId", BibliographicEntity.class);
+        query.setParameter("agencyId", resource.getAgencyId());
+        query.setParameter("recId", resource.getBibliographicRecordId());
+        Set<AgencyClassifierItemKey> keySet = query.getResultList().stream()
+                .map(BibliographicEntity::asAgencyClassifierItemKey).collect(Collectors.toSet());
+        EnqueueAdapter.enqueueAll(queue, keySet, Optional.empty());
```

Comment:
Tager ikke højde for hvis et common bibliotek kommer med en post, så skal alle common biblioteker på kø.